ISSUE: test_event_mode_fragments: 4 errors on missing references/roles/<role>/includes-events.yml

**Reported By**: skill-lead
**Severity**: Medium

**Found by**: skill-lead during #11042 investigation (cycle 1590, 2026-06-04).

## Symptom

`tests/test_event_mode_fragments.py::TestAc6M62ManifestWiring::test_role_manifest_includes_boot_bootstrap[<role>]` errors at setup for all four roles (worker, pm, verifier, dm) with:

```
AssertionError: missing manifest: D:\Dev\Dev\SquidSquad-2\references\roles\worker\includes-events.yml
```

(Same error for pm, verifier, dm.)

## Likely cause

Event-mode manifests (`includes-events.yml` per role) appear to have been removed at some point — either during the E6 V2 cutover (#10999, which retired v1 compose stack) or as part of the lazy-load contract rework (#9588). The test's setup fixture expects these files to exist; current cycle_pre output for skill shows `harness_status: unreachable` so polling mode is the only one used today, but the event-mode tests weren't deleted along with the manifests.

Confirm in either commit:
- `git log --oneline -- references/roles/worker/includes-events.yml`
- `git log --oneline -- references/roles/pm/includes-events.yml`

## Expected

If event-mode is permanently retired:
- Delete `TestAc6M62ManifestWiring` and any other test classes asserting `includes-events.yml` presence in this file.
- Delete any companion fragments under `references/sub-skills/common-events/` that the boot-bootstrap also drops (cross-check against #11045's TC-13 entry).

If event-mode is still planned (just dormant):
- Restore the four `includes-events.yml` files, or skip the test under `pytest.mark.skip` with a follow-up issue reference.

## Why this didn't ship with #11042

#11042 fixed stale-reference clusters that had a clear authoritative-state answer (files were intentionally removed, so prune installer-files.txt / delete tests). This cluster needs an event-mode roadmap decision (retired vs. dormant) before the test direction can be picked.

--- a/tests/test_event_mode_fragments.py
+++ b/tests/test_event_mode_fragments.py
@@ -158,17 +158,25 @@ class TestAc7TopicCoverage:
 
 
 class TestAc6M62ManifestWiring:
-    """AC-6 M-6.2 (post-#9588): the event-mode L1 base fragment + its four
-    supporting common-events fragments must reach the agent at runtime via
-    the boot bootstrap, NOT via compose-time inlining.
+    """AC-6 M-6.2 (post-#9588 + post-E6 cutover #10999): the event-mode L1
+    base fragment + its four supporting common-events fragments must reach
+    the agent at runtime via the boot bootstrap, NOT via compose-time
+    inlining.
 
     #9588 swapped the wiring mechanism: instead of every event-mode manifest
     listing all six common-events fragments and the template having matching
     `{{include:}}` directives, the manifest now contains `common/boot-bootstrap`
     only, and the bootstrap's content carries `Read references/sub-skills/
     common-events/<name>.md` directives that fire when the agent boots in
-    event mode. The previous compose-time-inline contract (#8915) is the old
-    contract; this class enforces the new one.
+    event mode.
+
+    #11046: post-E6 cutover (#10999, commit 1050bfe0) consolidated the v1
+    polling/event manifest split (``includes.yml`` + ``includes-events.yml``)
+    into a single mode-agnostic ``includes.yml`` per role. The per-role
+    ``includes-events.yml`` files are GONE — wake mode is a runtime
+    concern handled by ``common/boot-bootstrap`` at session start. This
+    fixture and the manifest test rebind from the retired file to the
+    surviving unified manifest. The wiring contract is unchanged.
     """
 
     REQUIRED_COMMON_EVENTS = [
@@ -184,11 +192,13 @@ class TestAc6M62ManifestWiring:
 
     @pytest.fixture(scope="class")
     def includes_by_role(self):
+        """#11046: switched from the retired ``includes-events.yml`` to the
+        post-cutover unified ``includes.yml``."""
         import yaml
         roles_dir = REPO_ROOT / "references" / "roles"
         out = {}
         for role in self.ROLES:
-            path = roles_dir / role / "includes-events.yml"
+            path = roles_dir / role / "includes.yml"
             assert path.exists(), f"missing manifest: {path}"
             data = yaml.safe_load(path.read_text(encoding="utf-8"))
             out[role] = data.get("includes", [])
@@ -212,7 +222,7 @@ class TestAc6M62ManifestWiring:
         are Read at runtime by the bootstrap instead."""
         includes = includes_by_role[role]
         assert "common/boot-bootstrap" in includes, (
-            f"references/roles/{role}/includes-events.yml must list "
+            f"references/roles/{role}/includes.yml must list "
             f"`common/boot-bootstrap` — #9588 made it the manifest entry "
             f"that gets the agent into event mode (or fallback polling)."
         )

